hub/apps/get-started/index.md

```diff
@@ -93,6 +93,30 @@ For more information about Windows Forms apps, see the following articles:
 * [API reference (.NET)](/dotnet/api/index)
 * [Enhancing Windows Forms apps](/dotnet/framework/winforms/advanced/)
 
+#### [React Native for Windows](#tab/rnw)
+[React Native](https://reactnative.dev) is a development platform from Facebook which allows building cross-platform apps.
+React Native for Windows brings React Native support for the Windows 10 and Windows 11 SDKs, enabling you touse JavaScript to build native Windows apps for all devices supported by Windows 10 and Windows 11, including PCs, tablets, 2-in-1s, Xbox, Mixed reality devices, etc. 
+
+With React Native for Windows, you write most or all of your app code in JavaScript - or TypeScript - and the framework produces a native UWP XAML application. If your app needs to call a platform API, you can usually do so through one of the many [community modules](https://reactnative.directory), or a module does not yet exist, you can easily write a native module to expose it.
+
+Here are some reasons to choose React Native for Windows:
+- you want to share code across platforms as much as possible, or you have web properties that you want to share code with.
+- improved developer productivity and inner loop, thanks to fast refresh.
+- your app's fundamentals (performance, accessibility, internationalization) are as good as a native UWP app.
+- you already have experience with JavaScript or TypeScript and feel comfortable
+- you can leverage all of the JS-only libraries on [npmjs.com](https://www.npmjs.com/), and many native libraries too.
+- On Windows, you get an app that "fits" with the platform, and has a standard and well designed way to allow you to call platform APIs, and write your own view managers or native modules.
+- Large and growing community momentum, with lots of community modules.
```

Comment:
This feels like the more appropriate place to link to https://reactnative.directory

Reply:
```suggestion
- Large and growing community momentum, with lots of [community modules](https://reactnative.directory).
```